### ISSUE TITLE
Additional metamagics optional rule

### ISSUE THREAD

**Issue by EfimEvgen** [chummer5a/chummer5a#5128]
**Is your feature request related to a problem? Please describe.**
There's option for "Additional Metamagics\Echoes" in Chummer's settings, and apparently a code to make it work.

**Describe the solution you'd like**
Can you add the optional rule to buy metamagics outside of initiations for additional karma? Same way enhancements work, I guess? Maybe with additional thing about what decides the max amount of additional metamagics (like, IG\Magic\chosen number etc). 

**Additional context**
That's coming from 4e optional rule; would be nice to be able to be able to access it without turning on ignoring the rules.
![image](https://github.com/chummer5a/chummer5a/assets/42835557/b4ef9324-c625-40b2-9d56-4e2e5820a389)

**Comment by JDub-76** [comment]
There even seems to be the option in chummer to set the karma cost for it
![image](https://github.com/chummer5a/chummer5a/assets/47260402/76999314-7dc0-4bac-b137-6c463991f25c)

**Comment by EfimEvgen** [comment]
Yes, this is the setting I refer to, and apparently there's some code exists for it, but it's not linked to anything.
